telegram/_message.py

```diff
@@ -2533,20 +2533,20 @@ def _parse_html(
                 elif entity.type == MessageEntity.URL and urled:
                     insert = f'<a href="{text}">{text}</a>'
                 elif entity.type == MessageEntity.BOLD:
-                    insert = '<b>' + text + '</b>'
+                    insert = f'<b>{text}</b>'
```

Comment:
can we move these changes to a standalone PR? they arent related to `Defaults.protected_content` … you can probably just cherry-pick the commit(s) to a new branch and revert them on this one